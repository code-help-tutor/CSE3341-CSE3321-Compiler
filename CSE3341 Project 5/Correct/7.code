procedure t7 is
	integer x;
	integer y;
	record a;
	record b;
begin
	x := 10;
	y := 1;
	while y < x do
		a := record b;
		a := new record[1];
		a[0] := 100;
		y := y+1;
	end
end
