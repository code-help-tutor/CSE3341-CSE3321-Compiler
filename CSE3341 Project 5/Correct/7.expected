gc:1
gc:0
gc:1
gc:0
gc:1
gc:0
gc:1
gc:0
gc:1
gc:0
gc:1
gc:0
gc:1
gc:0
gc:1
gc:0
gc:1
gc:0
